crates/swc/tests/tsc-references/checkJsxChildrenProperty12.1.normal.js

```diff
@@ -1,7 +1,7 @@
 //// [file.tsx]
 //! 
-//!   x Expected '>', got '{'
-//!     ,----
-//!  14 | return <InnerButton {...this.props} />
-//!     :                     ^
-//!     `----
+//!   x Import assignment cannot be used when targeting ECMAScript modules. Consider using `import * as ns from "mod"`, `import {a} from "mod"`, `import d from "mod"`, or another module format instead.
+//!    ,----
+//!  2 | import React = require('react');
```

Comment:
Maybe we should set `CommonJS` as default Module config.

Reply:
Yes.

https://github.com/swc-project/swc/blob/390b1d12e30a30b4791f493e34bfc8b6a790c85d/crates/swc/tests/tsc.rs#L112-L114

and

https://github.com/swc-project/swc/blob/390b1d12e30a30b4791f493e34bfc8b6a790c85d/crates/swc/tests/tsc.rs#L175-L177

Reply:
I'm fine with changing the default

Reply:
Hmmm. too many files changed. should not do it in this pull request.